src/SGB_Command_Sound.md

```diff
@@ -44,16 +44,21 @@ Used to transfer sound code or data to SNES Audio Processing Unit memory
  1-F   Not used (zero)
 ```
 
-The sound code/data is sent by VRAM-Transfer (4 KBytes).
+The sound code/data is sent by [VRAM transfer](<#VRAM Transfers>) as a contiguous list of "packets".
+
 All 16-bit values are little-endian.
 
+Data transfer packet format:
+```
+ 00-01  Size of data below (XX); if zero, this is instead a jump packet
+ 02-03  Destination address in S-APU RAM (typically $2B00, see below)
+ 04-XX  Data to be transferred
```

Comment:
Wait, I'm not so sure anymore. Does `XX` include the 4-byte header? Probably not; then `04-XX` should become `004-N+4`, or something like that.